src/Blocks/components/radio/radio.php

```diff
@@ -42,11 +40,9 @@ class="<?php echo esc_attr("{$componentClass}__input"); ?>"
 			name="<?php echo esc_attr($radioName); ?>"
 			id="<?php echo esc_attr($radioId); ?>"
 			value="<?php echo esc_attr($radioValue); ?>"
-			data-validation-required="<?php echo esc_attr($radioIsRequired); ?>"
 			data-tracking="<?php echo esc_attr($radioTracking); ?>"
 			<?php echo $radioIsChecked ? 'checked' : ''; ?>
 			<?php echo $radioIsDisabled ? 'disabled' : ''; ?>
```

Comment:
You can use https://developer.wordpress.org/reference/functions/checked/ and https://developer.wordpress.org/reference/functions/disabled/ for this 🙂

Reply:
tnx